translations/player_turns.tex/pl.po

```diff
@@ -180,16 +234,35 @@ msgid ""
 "If you would gain a second Negative Morale Token, you must instead \\textbf{discard your hand of Cards} the next time you end your Turn.\\par\n"
 "\n"
 msgstr ""
+"\\subsection*{Akcje Morale}\\index{Akcje Morale}\n"
+"Gracze mogą zyskać lub stracić Morale z powodu różnych efektów gry.\n"
+"Gdy zdobędziesz Morale, pobierz Żeton Wysokiego Morale\\index{Wysokie Morale} \\includesvg[height=10px]{\\svgs/morale_positive.svg}.\n"
+"Możesz posiadać tylko jeden taki Żeton.\n"
+"W momencie uzyskania drugiego Żetonu, możesz natychmiast wykorzystać pierwszy przed zdobyciem drugiego.\n"
+"Żeton Wysokiego Morale można wydać, aby wykonać dowolną z poniższych akcji w dowolnym momencie:\n"
+"\\begin{itemize}\n"
+"  \\item Pobierz kartę ze swojej Talii.\n"
+"  \\item Odrzuć dowolną ilość kart, a następnie dobierz tyle samo kart z Talii.\n"
+"  \\item Przerzuć jedną kość.\n"
+"\\end{itemize}\n"
+"Gdy twoje Morale obniży się, odrzuć Żeton Wysokiego Morale \\includesvg[height=10px]{\\svgs/morale_positive.svg} jeśli go posiadasz, w innym przypadku pobierz Żeton Niskiego Morale\\index{Niskie Morale} \\includesvg[height=10px]{\\svgs/morale_negative.svg}.\n"
+"Jeżeli natomiast twoje Morale podniesie się gdy masz Żeton Niskiego Morale, odrzuć ten Żeton.\n"
+"Gdy efekt gry każe ci dobrać drugi Żeton Niskiego Morale, zamiast tego pod koniec tury \\textbf{odrzuć wszystkie Karty z ręki}.\\par\n"
+"\n"
 
 #. type: multicols
 #: sections/player_turns.tex:87
 #, no-wrap
 msgid ""
-"\\note{5}{The Necropolis \\includesvg[height=10px]{\\svgs/necro-note.svg} Faction ignores any Morale effects.\n"
-"  They cannot ever gain or lose Morale for any reason.\n"
+"\\note{5}{The Necropolis \\includesvg[height=10px]{\\svgs/necro-note.svg} Faction ignores any Morale effects. \n"
+"They cannot ever gain or lose Morale for any reason.\n"
```

Comment:
```suggestion
"\\note{5}{The Necropolis \\includesvg[height=10px]{\\svgs/necro-note.svg} Faction ignores any Morale effects.\n"
"  They cannot ever gain or lose Morale for any reason.\n"
```
Wiesz co, utrzymujmy spójność białych znaków. To zmniejsza ryzyko dziwnych rzeczy przy składaniu dokumentu.